Formula/adoptopenjdk.rb

```diff
@@ -0,0 +1,29 @@
+class Adoptopenjdk < Formula
+  desc "Prebuilt binaries produced from OpenJDK class libraries"
+  homepage "https://adoptopenjdk.net/"
+  url "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jdk_x64_linux_hotspot_8u212b03.tar.gz"
+  version "8.0.212"
```

Comment:
```suggestion
  version "1.8.0.212"
```
Let's use `1.8.0.212`. Sorry for the back and forth. We're almost there.

Reply:
I got it. I change the format soon.